src/content/cloud/credentials.mdx

```diff
@@ -10,9 +10,9 @@ import Button from '@theme/Button';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-In this section will explain how to download your Kubernetes credentials and start developing your applications in Okteto  with your favorite CLI tools.
+This section explains how to download your Kubernetes credentials and start developing your applications in Okteto  with your favorite CLI tools.
 
-In order to do so, the first thing you need is to install Okteto CLI in your PATH. This is needed because the kubeconfig provided by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to authenticate you via Okteto to access your Kubernetes cluster. It is a common pattern used by many Kubernetes providers, such as Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), or Amazon Elastic Kubernetes Service (EKS) to connect to Kubernetes clusters in a more secure way.
+In order to do so, the first thing you need is to install Okteto CLI and ensure the executable is in your PATH. This is needed because the kubeconfig provided by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to authenticate access your Kubernetes cluster. It is a common pattern used by many Kubernetes providers, such as Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), or Amazon Elastic Kubernetes Service (EKS) to connect to Kubernetes clusters.
```

Comment:
```suggestion
The first thing you need to do is install the Okteto CLI and ensure the executable is in your PATH. This is needed because the kubeconfig provided by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to authenticate access your Kubernetes cluster. It is a common pattern used by many Kubernetes providers, such as Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), or Amazon Elastic Kubernetes Service (EKS) to connect to Kubernetes clusters.
```